common/api/imodel-components-react.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="react" />
-
```

Comment:
Any idea why this file was updated ? Does not seem to be related to your changes.

Reply:
api-extractor always does weird things with the ui packages. No one really understands why.

Reply:
I had to revert the changes to an appui package, I'll revert this one as well.

Reply:
Are these md files really used anywhere ?

Reply:
pmconne probably knows better, but maybe they're used in the docs generation? Unless that's what the csvs are for. Could also ask Will I believe.

Reply:
They make up the published API documentation.

Reply:
all changes to this file have been reverted, I am going to resolve this comment